pkg/serverless/invocationlifecycle/rand.go

```diff
@@ -1,7 +1,7 @@
 // Unless explicitly stated otherwise all files in this repository are licensed
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016 Datadog, Inc.
+// Copyright 2022-present Datadog, Inc.
```

Comment:
should it be ?
```suggestion
// Copyright 2016-present Datadog, Inc.
```

Reply:
Yes, that was an accident on my part. Will fix it 👍🏽